lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -343,13 +343,27 @@ def creds_search(*args)
 
     set_rhosts = false
     truncate = true
+    show_id = false
 
     cred_table_columns = [ 'host', 'origin' , 'service', 'public', 'private', 'realm', 'private_type', 'JtR Format', 'cracked_password' ]
     delete_count = 0
     search_term = nil
 
     while (arg = args.shift)
       case arg
+      # TODO: We should add this to other commands, other options: `-e` `--extended`
+      #   Or from sessions:
+      #           when "-x", "--list-extended"
+      #             show_extended = true
+      #           when "-v", "--list-verbose"
+      #             verbose = true
+      #   I think this has come up before too with the bruteforce PR that required adding Ids that we didn't render
+      #     https://github.com/rapid7/metasploit-framework/issues/17367
+      #     https://github.com/rapid7/metasploit-framework/pull/17601
+      #   There's also the new 'certs' command
+      when '--id'
+        cred_table_columns.unshift('id')
+        show_id = true
```

Comment:
Let's show the ID by default. This would align it with the `klist` and `certs` command to make it a bit more consistent.